contracts/staking/StakingV1_1.sol

```diff
@@ -618,6 +733,26 @@ contract StakingV1_1 is StakingState, StakingConstants {
         );
     }
 
+    function _addAltRewards(address token, uint256 amount) internal {
```

Comment:
Maybe we make this a public function that anyone can call?

Reply:
Actually you'd need a separate external function that do the transferFrom. Keep this one internal.

Reply:
mm, when sushi sends rewrds to staking contract we just cal this internal function to start new round, if we want to make it public we will need to update and add transferFrom assets from account to stakinng contract. But, yes I did it generic, so we can distribute some other rewards, not only sushi

Reply:
yes, lets make it public